activators/caila/src/main/kotlin/com/justai/jaicf/activator/caila/CailaIntentActivator.kt

```diff
@@ -53,7 +54,11 @@ class CailaIntentActivator(
                 }
             }
             // Sort all predicted intents by context relevance
-            .sortedByDescending { botContext.dialogContext.currentState.commonPrefixWith(it.first) }
+            .sortedByDescending {
```

Comment:
Now child states of the current state have the same "weights" as siblings. I suggest adding a comparison by context length for states with equal prefixes for increasing the "weight" of the child nodes.

Reply:
Actually, you can just replace comparison by prefix to comparison by context length (without `stepUp`), because nodes with lower common prefix length have lower context length.

Reply:
Unfortunately, I can't, because it will mess with global activators, where global activator if global activator is in different part of tree and has very long path.
I've changed a bit from PR and added fix for sibling selection. But this is still temporary and should be refactored with activators.